src/redux/user/user.sagas.js

```diff
@@ -273,6 +277,15 @@ export function* handleUserError(e) {
   }
 }
 
+export function* handleEmailError(e) {
```

Comment:
Looks like an additional state reducer and handler for email error is unnesesery

Reply:
fixed